pr-data.csv

```diff
@@ -4086,7 +4086,7 @@ https://github.com/visallo/vertexium,f1fd59e23610312cf0ec6756ea01baaf56cdb74e,ac
 https://github.com/visallo/vertexium,f1fd59e23610312cf0ec6756ea01baaf56cdb74e,inmemory,org.vertexium.inmemory.InMemoryGraphTest.manyVisibilities,NOD,,,
 https://github.com/visallo/vertexium,f1fd59e23610312cf0ec6756ea01baaf56cdb74e,inmemory,org.vertexium.inmemory.InMemoryGraphTest.testMarkPropertyVisible,NOD,,,
 https://github.com/visallo/vertexium,bc52f7cd33f6e559b3f144db77e160bddac0e003,core,org.vertexium.util.IncreasingTimeTest.currentTimeMillisReturnsTimeGreaterThanSystemTime,NOD,,,
-https://github.com/vmware/admiral,e4b02936cc7d4ff2714e7231db0c4373ba5d48a2,adapter/kubernetes,com.vmware.admiral.adapter.kubernetes.KubernetesMaintenanceTest.testAdapterRequestOnPeriodicMaintenance,NIO,Opened,https://github.com/vmware/admiral/pull/320,RepoArchived
+https://github.com/vmware/admiral,e4b02936cc7d4ff2714e7231db0c4373ba5d48a2,adapter/kubernetes,com.vmware.admiral.adapter.kubernetes.KubernetesMainteceTest.testAdapterRequestOnPeriodicMaintece,NIO,RepoArchived,https://github.com/vmware/admiral/pull/320,RepoArchived
```

Comment:
Test name was changed here, but the line should not be changed anyhow (do not change `Opened` to `RepoArchived`).